pkg/x509utils/convert.go

```diff
@@ -27,3 +29,21 @@ func ConvertPEMToDERs(certBytes []byte) ([][]byte, error) {
 		}
 	}
 }
+
+// ConvertPEMTox509Certs convert a PEM encoded certificate chain to a slice of x509 certificates.
+func ConvertPEMTox509Certs(certBytes []byte) ([]*x509.Certificate, error) {
```

Comment:
We should probably write a unit test for this.

Reply:
We should!

here it is: https://github.com/stackrox/stackrox/pull/12671/files#diff-3b7c044632b94fd50fdba586f0dee900dc0835fd01cfd34568c4c78334627f5d